.azure-pipelines/ci-build.yml

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - master
```

Comment:
OpenAPI.NET uses `vnext` branch as a default branch before `master`. We may want to have this flow in this repo as well.

Reply:
I'd prefer if we aligned all of them on main like the other repos we have at a later date if you don't mind?

Reply:
Leaving this as is will mean all merges to `master` will do a release. And this is the main branch for this repo.